frontend/cypress/e2e/dashboard.cy.js

```diff
@@ -13,73 +13,176 @@ describe("Pathology Dashboard", function () {
   it("User  Visits Pathology Dashboard", function () {
     homePage = loginPage.goToHomePage();
     dashboard = homePage.goToPathologyDashboard();
-
-    dashboard.checkForHeader("Pathology");
+    cy.wait(500);
   });
 
   it("User adds a new Pathology order", function () {
     homePage.goToOrderPage();
-    dashboard.addOrder("Histopathology");
+    dashboard.searchPatientByFName();
```

Comment:
Thanks @Bahati308 .
These steps looks duplicated. can you just define a generic function to addan oder with a program ??